### PR TITLE
AGENT-40: Use output kubeconfig to check for cluster complete

### PR DESCRIPTION
**Pull request by zaneb**

### CODE REVIEW

File: agent/05_agent_create_cluster.sh [hunk 1]
```diff
@@ -54,7 +54,7 @@ function wait_for_cluster_ready() {
   sleep 5m
```
Comment (andfasano) [inline agent/05_agent_create_cluster.sh:54]: I think that also this block could be removed, given that we don't have to wait anymore for the kubeconfig availabiity

Comment (zaneb) [reply agent/05_agent_create_cluster.sh:54]: I haven't tried but if we start oc before the bootstrap control plane is up, won't it fail?

Comment (andfasano) [reply agent/05_agent_create_cluster.sh:54]: Yes, you're right, we could keep the above block as a poor-man wait-for until the installer command will be ready

**Pull request merged**